3rdparty/libcxx/CMakeLists.txt

```diff
@@ -51,6 +51,11 @@ add_dependencies(libcxx libcxx_includes)
 set_target_properties(libcxx PROPERTIES
   POSITION_INDEPENDENT_CODE ON)
 
+# Configure clangw before turning off warnings.
+if (USE_CLANGW)
+  build_using_clangw(libcxx)
```

Comment:
So I was contemplating making `build_using_clangw` a no-op if `USE_CLANGW` is not true, so these three lines (repeated elsewhere) could just be:

```
maybe_cross_compile(libcxx)
```

If it's named `maybe_` then I think this is reasonable.

Reply:
Great suggestion.

Reply:
Fixed.